src/api/artifact/artifact.service.ts

```diff
@@ -81,6 +103,17 @@ const deleteArtifact = async (artifact_id: string): Promise<artifact> => {
   });
 };
 
+/**
+ * * Maps over an array of artifacts and adds a signed URL to each one
+ * @param {artifact[]} artifacts
+ */
+const addSignedUrlToArtifacts = (artifacts: artifact[]): ArtifactResponse[] => {
```

Comment:
Could probably make this function accept (artifact | artifact[]) and return appropriate object.
Preventing having to call it like addSignedUrlToArtifacts([artifact])[0] etc